static/app/components/group/streamlinedParticipantList.tsx

```diff
@@ -0,0 +1,113 @@
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+import {FocusScope} from '@react-aria/focus';
+
+import Avatar from 'sentry/components/avatar';
+import AvatarList from 'sentry/components/avatar/avatarList';
+import TeamAvatar from 'sentry/components/avatar/teamAvatar';
+import {Overlay, PositionWrapper} from 'sentry/components/overlay';
+import {t, tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Team, User} from 'sentry/types';
+import useOverlay from 'sentry/utils/useOverlay';
+
+interface DropdownListProps {
+  users: User[];
+  teams?: Team[];
+}
+
+export default function ParticipantList({users, teams}: DropdownListProps) {
+  const {overlayProps, isOpen, triggerProps} = useOverlay({
+    position: 'bottom-start',
+  });
+
+  const theme = useTheme();
+  const showHeaders = users.length > 0 && teams && teams.length > 0;
+
+  return (
+    <div>
+      <div {...triggerProps}>
+        <StyledAvatarList users={users} avatarSize={24} maxVisibleAvatars={3} />
+      </div>
+      {isOpen && (
+        <FocusScope contain restoreFocus autoFocus>
```

Comment:
I don't think this will actually do anything since there are no focusable elements in the menu afaict